packages/lodestar/test/e2e/chain/factory/block/assebleBlock.test.ts

```diff
@@ -34,6 +34,7 @@ import {
   StateRepository,
   VoluntaryExitRepository
 } from "../../../../../src/db/api/beacon/repositories";
+import { createIBeaconConfig } from "@chainsafe/eth2.0-config";
```

Comment:
you can use existing config from @chainsafe/eth2.0-config/lib/presets/minimal

Reply:
are you suggesting i just use that instead of sinon?

Reply:
yeah, unless you have some logic that depends on specific chain param